src/store.js

```diff
@@ -0,0 +1,14 @@
+import {createStore, applyMiddleware} from 'redux'
```

Comment:
Definitely ambitious to use Redux and axios for a 3 hour hackathon. Probably better for a simpler solution such as localStorage or firebase. Less to setup and handle for compressed time.